v2/thirdpartyemailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -266,18 +710,72 @@ SuperTokens.init({
     },
     recipeList: [
         ThirdPartyEmailPassword.init({
-            //highlight-start
-            getRedirectionURL: async (context) => {
-                // Prevent redirection after successful sign in/up by returning null.
-                if (context.action === "SUCCESS") {
-                    return null;
-                };
-            },
-            //highlight-end
-        })
+            // ...
+        }),
+        // ...
     ],
-})
+    // highlight-start
+    async getRedirectionURL(context) {
+        // The user will be taken to this path when they need to login.
+        if (context.action === "TO_AUTH") {
+            return "/auth?showModal=true"; // return the path where you are rendering the Auth UI
+        };
+    },
+    // highlight-end
+});
+
+function AuthPopup() {
+  let sessionContext = Session.useSessionContext();
+  const urlParams = new URLSearchParams(window.location.search);
+  const [isModalOpen, setIsModalOpen] = useState(urlParams.get("showModal") === "true");
+
+  const openModal = () => setIsModalOpen(true);
+  const closeModal = () => setIsModalOpen(false);
+
+  useEffect(() => {
+    if (sessionContext.loading) return;
+    if (sessionContext.doesSessionExist) {
+      closeModal();
+    }
+  }, [sessionContext]);
+
+  if (sessionContext.loading) return null;
+
+  return (
+    <div style={{ textAlign: "center" }}>
+      {sessionContext.doesSessionExist ? (
+        // We wrap this with <SessionAuth /> so that
+        // all claims are validated before showing the logged in UI.
+        // For example, if email verification is switched on, and
+        // the user's email is not verified, then <SessionAuth />
+        // will redirect to the email verification page.
+        <Session.SessionAuth>
+          <h2>You are logged In! </h2>
+          <h3>UserId: {sessionContext.userId}</h3>
+          <button onClick={() => Session.signOut()}>Sign Out</button>
+        </Session.SessionAuth>
+      ) : (
+        <div>
+          <h2>You are logged Out! </h2>
+          <h3>Open the modal below to Sign In</h3>
+          <button onClick={openModal}>Open Modal</button>
+        </div>
+      )}
+
+      <Modal
+        isOpen={isModalOpen}
+        onRequestClose={closeModal}
+        contentLabel="Auth Modal"
+      >
+        {/* highlight-next-line */}
+        <SignInAndUp />
+      </Modal>
+    </div>
+  );
+}
 ```
+</Answer>
+</Question>
 </TabItem>
 </FrontendPreBuiltUITabs>
 
```

Comment:
Remove the last section about "Optional: Rendering Auth components together in a popup and page"

Reply:
done